lua/terrortown/menus/gamemode/appearance/miscellaneous.lua

```diff
@@ -0,0 +1,10 @@
+--- @ignore
+
+CLGAMEMODESUBMENU.base = "base_gamemodesubmenu"
+
+CLGAMEMODESUBMENU.priority = 91
+CLGAMEMODESUBMENU.title = "submenu_appearance_miscellaneous_title"
+
+function CLGAMEMODESUBMENU:Populate(parent)
```

Comment:
Empty function?

Reply:
yes, because there are no misc settings as of yet. Same as it was before this move